src/main/java/mekanism/api/MekanismConfig.java

```diff
@@ -167,7 +168,6 @@ public static void setEntry(String type, boolean enabled)
 	public static class tools
 	{
 		public static double armorSpawnRate;
-		public static boolean enableTools = true;
```

Comment:
How come this is getting removed? Obsolete?

Reply:
its never used in any code